### ISSUE TITLE
Using ktlint - f remove wrong semi colon

### ISSUE THREAD

**Issue by Mathieu-Heuzey** [shyiko/ktlint#281]
Hello guys, i had an issue using the command "ktlint -f".

I have a kotlin class declaring a companion object like this "companion object;", I needed it to add more properties as extension function outside of it .

but when i use the command "ktlint -f" the semi colon is removed, and the my project does not compile because the companion object is not found anymore.

**Comment by Mathieu-Heuzey** [comment]
Here is my first class with my companion object

```
class FirstClass(private val param: param) {

    // Needed to add more properties as extension function outside of it
    companion object;

private var toto : Boolean = false

}
```

Here is my extension function 
```
fun FirstClass.Companion.create(c: Context) {
            return;
}
```

Then, when i run "ktlint -F", my semicolon after my companion object will be removed, and without it it does not compile. Is that clear for you or you need more than an example ?

**Comment by shyiko** [comment]
Thank you. That's all I need.

I'll have a fix shortly. As temporary workaround please use:

```
	companion object; // ktlint-disable no-semi
```

---

(for future reference (looks like kotlin compiler bug)):
```sh
$ cat FirstClass.kt
class FirstClass(private val param: String) {

    // Needed to add more properties as extension function outside of it
    companion object;

    private var toto: Boolean = false
}

fun FirstClass.Companion.create(c: Context) {
    return
}

class Context

$ ktlint FirstClass.kt 
/tmp/FirstClass.kt:4:21: Unnecessary semicolon

$ kotlinc FirstClass.kt 

$ ktlint -F FirstClass.kt 

$ kotlinc FirstClass.kt 
FirstClass.kt:9:16: error: unresolved reference: Companion
fun FirstClass.Companion.create(c: Context) {
               ^
```

**Comment by Mathieu-Heuzey** [comment]
thanks i will try the fix !

**Comment by shyiko** [comment]
Fixed in [0.29.0](https://github.com/shyiko/ktlint/blob/master/CHANGELOG.md#0290---2018-10-02).
